src/databricks/labs/ucx/source_code/known.py

```diff
@@ -103,13 +112,24 @@ def _cleanup_name(name):
     def rebuild(cls, root: Path):
         """rebuild the known.json file by analyzing the source code of installed libraries. Invoked by `make known`."""
         path_lookup = PathLookup.from_sys_path(root)
-        known_distributions = cls._get_known()
+        try:
```

Comment:
this is a good change :) let's limit this PR to it.

Reply:
Thanks; will do.